exec/java-exec/src/test/java/org/apache/drill/exec/physical/rowSet/impl/RowSetTestUtils.java

```diff
@@ -30,11 +29,9 @@ private RowSetTestUtils() { }
 
   public static List<SchemaPath> projectList(String... names) {
     List<SchemaPath> selected = new ArrayList<>();
-    for (String name: names) {
-
-      // Parse from string does not handle wildcards.
-
-      if (name.equals(SchemaPath.DYNAMIC_STAR)) {
+    for (String name : names) {
+      if (name.equals(SchemaPath.DYNAMIC_STAR) ||
+          name.equals("*")) {
```

Comment:
As far as I understand start column is replaced with `**` in Calcite, do we need to check for `*` as well?